debian/rules

```diff
@@ -13,4 +13,5 @@ override_dh_auto_configure:
 	  -DCMAKE_BUILD_TYPE=RelWithDebInfo \
 	  -DJPEGXL_FORCE_SYSTEM_GTEST=ON \
 	  -DJPEGXL_FORCE_SYSTEM_BROTLI=ON \
-	  -DJPEGXL_FORCE_SYSTEM_HWY=ON
+	  -DJPEGXL_FORCE_SYSTEM_HWY=ON \
+	  -DJPEGXL_ENABLE_PLUGINS=ON 
```

Comment:
I think it is about time to enable JPEGXL_ENABLE_GIMP_SAVING by default, but probably in the cmakefile, right? (not related to this PR).

Reply:
That flag was actually removed, so not a concern :)